src/main/scala/org/clulab/coref/Coref.scala

```diff
@@ -347,11 +347,9 @@ class Coref {
 
       if (mentions.isEmpty) Nil
       else {
-        if (debug) {
-          println("BEFORE COREF")
-          displayMentions(mentions, mentions.head.document)
-          println("Starting coref...")
-        }
+        logger.debug("BEFORE COREF")
+        displayMentions(mentions, mentions.head.document)
```

Comment:
Is this now being called all the time?
`displayMentions` uses `println` to display the mentions.
We should use some other option that returns the string to print and sent it to the logger, so that we can turn it off.
Currently we are printing more than 4mb of stuff, and travis doesn't like that.